### PR TITLE
Center toggle of video-toggle

### PR DESCRIPTION
**Pull request by MiepHD**
Appended video-toggle to `#main-panel` so it can be centered over the video.
Centered it through setting left to `50% - 120px` and set position of `#main-panel` to relative that it uses this to center.

### CODE REVIEW

File: plugins/video-toggle/button-switcher.css [hunk 1]
```diff
@@ -18,6 +22,7 @@
     color: #fff;
     padding-right: 120px;
     position: absolute;
+    left: calc(50% - 120px);
```
Comment (th-ch) [inline plugins/video-toggle/button-switcher.css]: It works but (opinionated) I preferred having it on the side, it covers less the video 😄 Maybe we can have it as an additional css file, and make it a plugin option to center or not the toggle? This way users have a choice

**Comment by MiepHD** [comment]
You can now choose between left, middle and right

**Review by th-ch — APPROVED** [review]

**Comment by Araxeus** [comment]
This can possibly break the plugin (and it did for me, multiple time
`Cannot read properties of undefined.`
`mainpanel.style.setProperty("--align", "0px");`

this whole code block needs to be inside an `apiLoaded` closure, we can't guarantee that `$('#main-panel')` exists when this code is called

In other words, instead of being inside the default export function, it should be inside the `setup` function which is called only when the page is loaded

@MiepHD @th-ch

**Comment by MiepHD** [comment]
Hm.. that's right. I'll correct that.

**Pull request merged**